lib/heuristics/scheduling_heuristic.rb

```diff
@@ -86,7 +86,7 @@ def initialize(vrp, job = nil)
 
     def compute_initial_solution(vrp, &block)
       if @services_data.empty?
-        empty_result(vrp)
+        vrp[:preprocessing_heuristic_result] = Wrappers::Wrapper.new.empty_result('heuristic', vrp)
```

Comment:
As the wrapper object created here with new is never used later. We should have to create an new object but Wrappers::Wrapper.empty_result directly

Reply:
For now, scheduling does not inherit from Wrapper::Wrapper, I think we need to do it like this then. Do you agree ?